ClientApp/src/app/services/indicator/indicator.service.ts

```diff
@@ -14,6 +15,7 @@ export class IndicatorService {
     public static BASE_URL = `api/Indicators`;
 
     public static INDICATORS_API = '/api/Indicators/';
+    public static ADD_REGISTRY_METHOD = "/AddRegistry";
```

Comment:
Add a registry must work on POST of the registry API `/api/registries`
The POST verb is the create CRUD on a HTML REST